Common/syntheaPostInstall.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+mkdir /tmp/loader
+pushd /tmp/loader
+curl -fsSL --progress-bar https://github.com/OSEHRA/VistA-FHIR-Data-Loader/releases/download/0.1/VISTA_SYNTHETIC_DATA_LOADER_BUNDLE_0P2T1.KID.zip -o loader.zip
+unzip -q loader.zip -d /tmp/loader/Synthea
+echo "Installing Synthea ingestor patch"
+system=1
+instanceName='-cn $instance'
+if ($installgtm || $installYottaDB); then
+  system=2
+  instanceName=''
+fi
+if [ ! -d "VistA" ]; then
```

Comment:
If i read this right we are in the /tmp/loader directory and looking for the VistA directory, I highly doubt that will ever exist in new installs, is there anywhere else that we download the VistA test repo and check for that instead?